openapi.yaml

```diff
@@ -525,343 +605,340 @@ parameters:
-        example: '2018-12-31'
+        example: 1970-1-21
       beginDate:
         type: string
         format: date
         description: Begin date of job.
-        example: '2018-12-31'
+        example: 1970-1-21
       endDate:
         type: string
         format: date
         description: End date of job.
-        example: '2019-01-01'
-      accruesLeave:
+        example: 1971-2-15
+      accruesLeaveInd:
         type: boolean
-        description: If true, the job accrues leave.
+        description: If true, the job accrues leave time.
       contractBeginDate:
         type: string
         format: date
         description: The date the contract begins.
-        example: '2018-12-31'
+        example: 1970-1-21
       contractEndDate:
         type: string
         format: date
         description: The date the contract ends.
-        example: '2018-12-31'
+        example: 1971-2-21
       contractType:
         type: string
-        description: Contract type code/ID.
-      locationID:
+        enum: [P, S, O]
+        description: |
+          Contract type code/Id.
+          `P` - Primary
+          `S` - Secondary
+          `O` - Overload
+      locationId:
         type: string
-        description: Location ID of where job will be performed.
+        description: Location Id of where job will be performed.
       status:
         type: string
-        description: Job status
+        description: Job status.
         enum:
           - Active
           - Leave without pay but with benefits
           - Leave without pay and benefits
           - Leave with full pay and benefits
           - Leave with partial pay and benefits
           - Terminated
-        example: Active
       description:
         type: string
-        description: Description of job
+        description: Description of job.
       personnelChangeDate:
         type: string
         format: date
         description: Date the last change was made to the job record.
-        example: '2018-12-31'
+        example: 1970-1-1
       changeReasonCode:
         type: string
         description: The reason the last change was made to the job record.
       fullTimeEquivalency:
         type: number
         format: float
-        description: Full time eqivalency where full time = 1
-        example: 0.49
+        description: Full time equivalency where full time = 1.
+        example: 0.25
       appointmentPercent:
         type: number
         format: float
         description: FTE presented as a whole percentage.
-        example: 49
+        example: 25
       salaryStep:
         type: string
         format: integer
-        example: '1'
+        example: 1
       salaryGroupCode:
         type: string
+        description: Salary group code for the job.
+        example: 12345A
       strsAssignmentCode:
         type: string
-      supervisorOsuID:
+        description: STRS assignment code for the job.
+        example: 12
```

Comment:
If the value can be parsed as a number, it should be quoted: https://github.com/osu-mist/style-guides/blob/master/OpenAPI.md#quoting-conventions